utils/post.js

```diff
@@ -156,14 +156,15 @@ const modifyAnonymousAndBlockPost = async (
   return feedWithAnonymous;
 };
 
-const modifyAnonimityPost = (item) => {
+const modifyAnonimityPost = async (item, isBlurredPost = true) => {
   const newItem = {...item};
 
   if (newItem.anonimity) {
     newItem.actor = {};
     newItem.to = [];
     newItem.origin = null;
     newItem.object = '';
+    newItem.isBlurredPost = isBlurredPost;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyAnonimityPost` function has been modified to include an optional parameter `isBlurredPost`. However, the `isBlurredPost` property is being set inside the condition that checks for `newItem.anonimity`. This means that if a post is not anonymous, it will never be blurred even if `isBlurredPost` is true. If the intention is to allow any post (anonymous or not) to be blurred, then the assignment of `isBlurredPost` should be outside the `if` block.

```diff
-  if (newItem.anonimity) {
-    newItem.actor = {};
-    newItem.to = [];
-    newItem.origin = null;
-    newItem.object = '';
-    newItem.isBlurredPost = isBlurredPost;
-  }
+  newItem.isBlurredPost = isBlurredPost;
+  if (newItem.anonimity) {
+    newItem.actor = {};
+    newItem.to = [];
+    newItem.origin = null;
+    newItem.object = '';
+  }

   return newItem;
```
This change ensures that the `isBlurredPost` property is always set, regardless of whether the post is anonymous or not.


<!-- This is an auto-generated comment by OSS CodeRabbit -->